src/back/routes/badges.ts

```diff
@@ -4,17 +4,20 @@ import handleAPI from 'decentraland-gatsby/dist/entities/Route/handle'
 import routes from 'decentraland-gatsby/dist/entities/Route/routes'
 import { Request } from 'express'
 
-import { UserBadges, toOtterspaceRevokeReason } from '../../entities/Badges/types'
+import { ActionStatus, UploadResult, UserBadges, toOtterspaceRevokeReason } from '../../entities/Badges/types'
+import { mintBadge, storeBadgeSpec } from '../../entities/Badges/utils'
 import isDebugAddress from '../../entities/Debug/isDebugAddress'
 import { BadgesService } from '../../services/BadgesService'
 import { AirdropOutcome } from '../models/AirdropJob'
-import { validateAddress } from '../utils/validations'
+import { validateAddress, validateDate, validateRequiredString, validateRequiredStrings } from '../utils/validations'
 
 export default routes((router) => {
   const withAuth = auth()
   router.get('/badges/:address/', handleAPI(getBadges))
-  router.post('/badges/airdrop/', withAuth, handleAPI(airdropBadges))
-  router.post('/badges/revoke/', withAuth, handleAPI(revokeBadge))
+  router.post('/badges/airdrop/', withAuth, handleAPI(airdrop))
+  router.post('/badges/revoke/', withAuth, handleAPI(revoke))
+  router.post('/badges/upload/', withAuth, handleAPI(upload))
+  router.post('/badges/mint/', withAuth, handleAPI(mint))
```

Comment:
What about renaming upload to `upload-spec` with function name uploadBadgeSpec or uploadSpec?
And the same with the mint one.